releasenotes/notes/add-kubelet-sli-metrics-7ba74568b16ca36e.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Adds support for kubelet SLI metrics
```

Comment:
maybe you can add the name of the new metrics in the description

Reply:
however, but I'm wondering if we should add a changelog, because the kubelet golang check is not yet released so it can be misleading for user.

Reply:
That makes sense. Should I add the no changelog label then and add this note when I do the Python implementation?